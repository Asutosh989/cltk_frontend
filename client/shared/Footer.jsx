Footer = React.createClass({
  render(){
		let date = new Date();
		let year = date.getFullYear();
    return (
			<footer className="bg-dark">
		    <div className="container">
		      <div className="row">
		        <div className="col-sm-12 text-center">
		          <h3 className="logo">Classics Archive</h3>
		          <ul className="list-inline social-list mb0">
		            <li>
									<a className="md-button md-ink-ripple" href="http://github.com/cltk">
										<i className="mdi mdi-github-circle"></i>
										<div className="md-ripple-container"></div>
									</a>
		            </li>
		            <li>
									<a className="md-button md-ink-ripple" href="http://twitter.com/@classicsarchive">
										<i className="mdi mdi-twitter"></i>
										<div className="md-ripple-container"></div>
									</a>
		            </li>
		            <li>
									<a className="md-button md-ink-ripple" href="http://plus.google.com/+ClassicsArchive">
										<i className="mdi mdi-google-plus"></i>
										<div className="md-ripple-container"></div>
									</a>
		            </li>
		          </ul>
		        </div>
					</div>
		      <div className="row">
		        <div className="col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2 text-center">
		          <div className="footer-links">
								<a className="md-button md-ink-ripple" href="/">
									<span>Home</span>
									<div className="md-ripple-container"></div>
								</a>
								<a className="md-button md-ink-ripple" href="http://api.cltk.org/browse">
									<span>Read</span>
									<div className="md-ripple-container"></div>
								</a>
								<a className="md-button md-ink-ripple" href="http://api.cltk.org/search">
									<span>Search</span>
									<div className="md-ripple-container"></div>
								</a>
								<a className="md-button md-ink-ripple" href="/">
									<span>Contribute</span>
									<div className="md-ripple-container"></div>
								</a>
							</div>
		          <span className="fade-1-4">Copyright the Classical Languages ToolKit, {year}.  All of the media presented on this site are available through the Creative Commons Attribution 4.0 International, Free Culture License.</span>

		        </div>
		      </div>
		    </div>
		  </footer>
		);
	}
});
