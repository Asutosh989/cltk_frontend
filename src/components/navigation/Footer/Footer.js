import React from 'react';
import {
  Grid,
  Row,
  Col,
} from 'react-bootstrap';
import { Link } from 'react-router';

import './Footer.css';

export default class Footer extends React.Component {
	render() {
		const now = new Date();
		const year = now.getFullYear();
		return (
			<section id="footer">
				<Grid>
					<Row>
						<Col lg={4}>
							<span className="footerCopyright">
                Â© cltk {year}
							</span>
							<span className="footerBuiltBy">
                See more at
								<a
									href="http://cltk.org"
									target="_blank"
									rel="noopener noreferrer"
								>
                  cltk.org
								</a>
								.
							</span>
						</Col>
						<Col lg={8}>
							<div className="footerLinks">
								<Link href="mailto:contact@cltk.org">
									Contact
								</Link>
							</div>
						</Col>
					</Row>
				</Grid>
			</section>
		);
	}
}
