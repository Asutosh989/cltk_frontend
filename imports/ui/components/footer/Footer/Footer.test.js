
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import Footer from './Footer';

describe('Footer', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<Footer />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
