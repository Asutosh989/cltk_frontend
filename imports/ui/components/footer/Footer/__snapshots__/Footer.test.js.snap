// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="bg-dark"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8 col-md-offset-2 col-sm-9 col-sm-offset-1 text-center"
      >
        <h3
          className="logo"
        >
          CLTK Archive
        </h3>
        <div
          className="footer-nav"
        >
          <a
            disabled={false}
            href="/"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
                "zIndex": 1,
              }
            }
            tabIndex={0}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                HOME
              </span>
            </div>
          </a>
          <a
            disabled={false}
            href="/browse"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
                "zIndex": 1,
              }
            }
            tabIndex={0}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                READ
              </span>
            </div>
          </a>
          <a
            disabled={false}
            href="/search"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
                "zIndex": 1,
              }
            }
            tabIndex={0}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                SEARCH
              </span>
            </div>
          </a>
          <a
            disabled={false}
            href="http://github.com/cltk/cltk"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            rel="noopener noreferrer"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
                "zIndex": 1,
              }
            }
            tabIndex={0}
            target="_blank"
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                CONTRIBUTE
              </span>
            </div>
          </a>
          <a
            disabled={false}
            href="//cltk.org/"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            rel="noopener noreferrer"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
                "zIndex": 1,
              }
            }
            tabIndex={0}
            target="_blank"
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                CLTK.ORG
              </span>
            </div>
          </a>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-12 text-center"
      >
        <ul
          className="list-inline social-list mb0"
        >
          <li>
            <a
              disabled={false}
              href="http://github.com/cltk"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseOut={[Function]}
              onTouchStart={[Function]}
              onTouchTap={[Function]}
              style={
                Object {
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "background": "none",
                  "border": 10,
                  "boxSizing": "border-box",
                  "cursor": "pointer",
                  "display": "inline-block",
                  "fontFamily": "Roboto, sans-serif",
                  "fontSize": 0,
                  "fontWeight": "inherit",
                  "height": "auto",
                  "margin": 0,
                  "muiPrepared": true,
                  "outline": "none",
                  "overflow": "visible",
                  "padding": 12,
                  "position": "relative",
                  "textDecoration": "none",
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                  "verticalAlign": "middle",
                  "width": "auto",
                  "zIndex": 1,
                }
              }
              tabIndex={0}
            >
              <div
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
              >
                <span
                  className="mdi mdi-github-circle"
                  color="rgba(0, 0, 0, 0.87)"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </a>
          </li>
          <li>
            <a
              disabled={false}
              href="http://twitter.com/@cltkarchive"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseOut={[Function]}
              onTouchStart={[Function]}
              onTouchTap={[Function]}
              style={
                Object {
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "background": "none",
                  "border": 10,
                  "boxSizing": "border-box",
                  "cursor": "pointer",
                  "display": "inline-block",
                  "fontFamily": "Roboto, sans-serif",
                  "fontSize": 0,
                  "fontWeight": "inherit",
                  "height": "auto",
                  "margin": 0,
                  "muiPrepared": true,
                  "outline": "none",
                  "overflow": "visible",
                  "padding": 12,
                  "position": "relative",
                  "textDecoration": "none",
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                  "verticalAlign": "middle",
                  "width": "auto",
                  "zIndex": 1,
                }
              }
              tabIndex={0}
            >
              <div
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
              >
                <span
                  className="mdi mdi-twitter"
                  color="rgba(0, 0, 0, 0.87)"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </a>
          </li>
          <li>
            <a
              disabled={false}
              href="http://plus.google.com/+cltkarchive"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseOut={[Function]}
              onTouchStart={[Function]}
              onTouchTap={[Function]}
              style={
                Object {
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "background": "none",
                  "border": 10,
                  "boxSizing": "border-box",
                  "cursor": "pointer",
                  "display": "inline-block",
                  "fontFamily": "Roboto, sans-serif",
                  "fontSize": 0,
                  "fontWeight": "inherit",
                  "height": "auto",
                  "margin": 0,
                  "muiPrepared": true,
                  "outline": "none",
                  "overflow": "visible",
                  "padding": 12,
                  "position": "relative",
                  "textDecoration": "none",
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                  "verticalAlign": "middle",
                  "width": "auto",
                  "zIndex": 1,
                }
              }
              tabIndex={0}
            >
              <div
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
              >
                <span
                  className="mdi mdi-google-plus"
                  color="rgba(0, 0, 0, 0.87)"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </a>
          </li>
        </ul>
        <span
          className="copyright-information fade-1-4"
        >
          Copyright Classical Language ToolKit, 
          2017
          .  All of the media presented on this site originating from the CLTK are available through the Creative Commons Attribution 4.0 International, Free Culture License. Media originating from other sources are specific to the contributor. Review specific corpora information here: 
          <a
            href="https://github.com/cltk"
            target="_blank"
          >
            https://github.com/cltk
          </a>
          .
        </span>
      </div>
    </div>
  </div>
</footer>
`;
